plugins/modules/packaging/os/openbsd_pkg.py

```diff
@@ -255,7 +266,7 @@ def package_present(names, pkg_spec, module):
 
 # Function used to make sure a package is the latest available version.
 def package_latest(names, pkg_spec, module):
-    if module.params['build'] is True:
+    if module.params['build'] is True and module.params['state'] == 'latest':
```

Comment:
This change seems unnecessary. We only call package_latest() if the state is set to "latest", so checking it again just after entering the function is redundant.
```suggestion
    if module.params['build'] is True:
```

Reply:
@sizeofvoid if you back out that change I think we are ready to merge, ill do a final check after it has been fixed but I think it is looking good otherwise.